### PR TITLE
Update index.md

### LINKED ISSUE #25054: Missing if statement for "Escape" key
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_build_custom_form_controls

### What specific section or headline is this issue about?

Handling the control's value

### What information was incorrect, unhelpful, or incomplete?

'keyup' event handler, lines 650 - 664 on the [following](https://github.com/mdn/content/blob/main/files/en-us/learn/forms/how_to_build_custom_form_controls/index.md?plain=1), is missing an if statement for the escape key. This is included in the source code for example 4 but is omitted from the corresponding code within the article. 

### What did you expect to see?

Consistency between the example 4 source code and the corresponding code within the article. 

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/forms/how_to_build_custom_form_controls`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_build_custom_form_controls
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/forms/how_to_build_custom_form_controls/index.md
* Last commit: https://github.com/mdn/content/commit/8e2641ebe076ab89299c77a51ece882de4ba5efb
* Document last modified: 2023-02-23T23:55:46.000Z

</details>

### PR DESCRIPTION
**Pull request by Phil-G-94**
Add in if statement on lines 652-656 to keep article code consistent with it's example 5 source code counterpart.
Fixes #25054.

### CODE REVIEW

File: files/en-us/learn/forms/how_to_build_custom_form_controls/index.md [hunk 1]
```diff
@@ -649,6 +649,11 @@ window.addEventListener('load', () => {
     // Each time a user uses their keyboard on a focused control, we update the value accordingly
     select.addEventListener('keyup', (event) => {
       let index = getIndex(select);
+
```
Comment (dawei-wang) [inline files/en-us/learn/forms/how_to_build_custom_form_controls/index.md:652]:
```suggestion

```

**Review by dawei-wang — COMMENTED** [review]
linting

**Review by dawei-wang — COMMENTED** [review]
Hi @Phil-G-94, This LGTM. I don't have the rights to merge this, so I'll send this back to @bsmth for final review.

**Review by bsmth — APPROVED** [review]
Thanks @Phil-G-94 for the addition!

The order [is slightly different](https://github.com/mdn/content/blob/863ac9a285983f5cfa534307ce080506f4fbb43b/files/en-us/learn/forms/how_to_build_custom_form_controls/example_4/index.md?plain=1#L268-L282) in case someone notices in future, but functionally, it's good.

Thanks @dawei-wang for flagging 🙌🏻

**Comment by Phil-G-94** [comment]
Thank you for the help @dawei-wang, and the approval and merge @bsmth. 

Noted re slightly different order, apologies for the oversight. Rookie contributor 👨🏼‍🎓

**Pull request merged**